tests/cypress/integration/actions_tasks3/case_107_connected_file_share.js

```diff
@@ -0,0 +1,87 @@
+        cy.get('#name').type(taskName);
+        cy.addNewLabel(labelName);
+        cy.contains('[role="tab"]', 'Connected file share').click();
+        cy.get('.cvat-share-tree').should('be.visible').within(() => {
+            cy.get('[aria-label="plus-square"]').click();
+            cy.get('[title]').should('have.length', 4) // Also "root"
+            cy.exec('docker exec -i cvat /bin/bash -c "ls ~/share"').then((command) => {
+                stdoutToList = command.stdout.split('\n');
+                // [image_case_107_1.png, image_case_107_2.png, image_case_107_3.png]
+                expect (stdoutToList.length).to.be.eq(3);
+                stdoutToList.forEach((el) => {
+                    cy.get(`[title="${el}"]`).should('exist');
+                    // Click on the checkboxes
+                    cy.get(`[title="${el}"]`).prev().click().should('have.attr', 'class').and('contain', 'checked');
+                });
+            });
+        });
+        cy.contains('button', 'Submit').click();
+        cy.get('.cvat-notification-create-task-success').should('exist').find('button').click();
+        cy.get('.cvat-notification-create-task-success').should('exist').find('[data-icon="close"]').click();
+        cy.get('.cvat-task-details').should('exist');
+    }
+
+    before(() => {
+        cy.visit('auth/login');
+        cy.login();
+    });
+
+    afterEach(() => {
+        cy.goToTaskList();
+        cy.deleteTask(taskName);
+    });
+
+    after(() => {
+        // Renaming to the original name
+        cy.exec(`mv ${assetLocalPath}/${stdoutToList[0]}.bk ${assetLocalPath}/${stdoutToList[0]}`);
+    })
+
+    describe(`Testing case "${caseId}"`, () => {
+        it('Create a task with "Connected file share".', () => {
+            createOpenTaskWithShare();
+            cy.openJob();
+            cy.get('.cvat-player-filename-wrapper').then((playerFilenameWrapper) => {
+                for (let el = 0; el < stdoutToList.length; el++) {
+                    cy.get(playerFilenameWrapper).should('have.text', stdoutToList[el]);
+                    cy.checkFrameNum(el);
+                    cy.get('.cvat-player-next-button').click().trigger('mouseout');
+                }
+            });
+        });
+
+        it('Check "Fix problem with getting cloud storages in Firefox".', () => {
+            cy.goToTaskList();
+            createOpenTaskWithShare();
+            // Rename the image
+            cy.exec(`mv ${assetLocalPath}/${stdoutToList[0]} ${assetLocalPath}/${stdoutToList[0]}.bk`)
+                .then((fileRenameCommand) => {
+                    expect(fileRenameCommand.code).to.be.eq(0);
+                });
+            cy.exec(`docker exec -i cvat /bin/bash -c "find ~/share -name "*.png" -type f"`)
+                .then((findFilesCommand) => {
+                    // [image_case_107_2.png, image_case_107_3.png]
+                    expect(findFilesCommand.stdout.split('\n').length).to.be.eq(2);
+                });
```

Comment:
Why need this check if the previous command
```
cy.exec(`mv ${assetLocalPath}/${stdoutToList[0]} ${assetLocalPath}/${stdoutToList[0]}.bk`)
                .then((fileRenameCommand) => {
                    expect(fileRenameCommand.code).to.be.eq(0);
                });
``` 
will return code 0?

Reply:
To make sure that the file has been renamed. And there are only 2 images used in the task. Checking the exit code of the rename command has been added just in case.